docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+# Introduction to image-based Linux distributions from Fedora and Universal Blue
+
+Image-based in the context of Linux distributions means that the operating system is packaged as a sort of image and everything added by the user is considered a separate layer. In image-based distributions the root filesystem is mounted as read-only while the OS is booted. This means that all the changes to the base system are only applied after a reboot. Fedora uses [`ostree`](https://ostreedev.github.io/ostree/) (like git for the filesystem) for its atomic filesystem updates and [`rpm-ostree`](https://docs.fedoraproject.org/en-US/fedora/latest/system-administrators-guide/package-management/rpm-ostree/) (specialized equivalent of Fedora's `dnf`) for the image and package management system.
+
+## Isn't this just another immutable Linux distribution?
```

Comment:
We should explain the terminology better here in a way that does not leave the reader confused. We should mention that "immutable" is a more common term and why we prefer something else. This section is currently a lacking draft. 

If someone wants to try this out, they'd be welcome to. I'm also open to the smallest and largest of suggestions here.

Reply:
I don't think it's worth mentioning immutable Linux distros at all on the introduction page. This feels more like an FAQ question than anything else

Reply:
Sure, this could be moved to the FAQ. I just thought that people familiar with Linux would have heard of immutable distributions and think "wait isn't this just that".